iothub_client/src/iothub_client_core_ll.c

```diff
@@ -633,21 +637,15 @@ static const char* IoTHubClientCore_LL_GetModelId(void* ctx)
     return result;
 }
 
-static bool IoTHubClientCore_LL_MessageCallbackFromInput(MESSAGE_CALLBACK_INFO* messageData, void* ctx)
+static bool IoTHubClientCore_LL_MessageCallbackFromInput(IOTHUB_MESSAGE_HANDLE messageHandle, void* ctx)
 {
     bool result;
     IOTHUB_CLIENT_CORE_LL_HANDLE_DATA* handleData = (IOTHUB_CLIENT_CORE_LL_HANDLE_DATA*)ctx;
```

Comment:
Nit - because we have two handles now - the message handle and core_ll_handle - I think `handleData` is ambiguous.  I'd think something like `clientHandleData` for this variable?

Similiar feedback in `IoTHubClientCore_LL_MessageCallback`.

Reply:
Makes sense. Done.